### DISCUSSION TITLE
Kaç puanınız var ve Nasıl fazla puan kasabiliri?????

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#630] [Soru Cevap - Türkçe] [answered]
İnsanlar bu kadar İbax puanı  kasıyor? bilmiyorum dağıtım nasıl olacak oda bi haber bilgisi olan varsa Yazarsa minnettar kalırız :)

**Comment by ghost** [comment, marked as answer]
Bilenlerde söylemez kanka

**Reply by Garcanlar28** [reply]
1 subatta dahitim var gorevleri yapip biriktiriyorsun

**Reply by Garcanlar28** [reply]
https://t.me/AirdropsVeDapps

**Reply by Garcanlar28** [reply]
buraya gel yardimci olayim

**Reply by Vedatsincan** [reply]
Ot

**Reply by kadir151** [reply]
burdan puan geliyormu

**Comment by fuinonfui** [comment]
bu github ı nasıl kullanacağız burdan daha fazla puan almak için bilgi vermeniz mümkün mü?

**Reply by slertolga** [reply]
+1

**Reply by Skips1** [reply]
Kanka çarpıcı konulara deyinmen lazım ve tartışmalara katılman lazım  :) Bunun la ilgili bir yazı yazdım sonunda bakabilirsin

**Comment by cryptopus58** [comment]
Github konusunda bilgi verebilecek biri var mı
Burada nasıl puan kazanıyorsunuz

**Reply by Stendardo17** [reply]
Bunun cevabını yarın sizlere açıklayacağım. Beni takipte kalın. Şu an çok uykum var ve anlatımım şu an karışık olabilir. Yarın sorularınızı eksiksiz olarak size yanıtlayacağım.

**Reply by zukenberger** [reply]
Kanklar github ön plada olmasının nedeni bu platform daha çok yazılım ve kodlamalar üzerine. İbax network kendi açıklarını bulmak kendi platformu yazılım ve kodlamalar sayesinde daha da geliştirmek için github üzerinden iş yürütüyor. Kısacası yazılım ve kodlama bilenler 5-0 önde.

**Comment by tahir0007** [comment]
nedir bu necidir :)

**Comment by Byjokers** [comment]
> İNsablar bu kadar İbax puanı kasıyor hakkında bilgilendirme nasıl olacak oda bi haber bilgisi olan varsa yazarsa kalırız :)

Bende 12 k var 😔

**Comment by sysdba** [comment]
Eğlenceli görünüyor

**Reply by Garcanlar28** [reply]
https://t.me/AirdropsVeDapps

**Reply by Garcanlar28** [reply]
buraya gel yardimci olcam

**Comment by Vedatsincan** [comment]
Benim şimdi ki puanım 25000 dir bu sabahta yeni twt geldi 2000 toplam 25000
dir

26 Oca 2022 Çar ÖS 1:13 tarihinde xPunchline ***@***.***>
şunu yazdı:
